lib/bloom_remit_client/responses/get_deposit_targets_response.rb

```diff
@@ -0,0 +1,14 @@
+module BloomRemitClient
+  class GetDepositTargetsResponse < BaseResponse
+
+    attribute :deposit_targets, Array, lazy: true, default: :default_deposit_targets
+
+    def default_deposit_targets
+      JSON.parse(body).map do |deposit_target_hash|
```

Comment:
You can avoid duplication by just using `parsed_body` (assuming it exists already)
```suggestion
      parsed_body.map do |deposit_target_hash|
```